packages/rosaenlg/src/LanguageGerman.ts

```diff
@@ -134,6 +134,7 @@ export class LanguageGerman extends LanguageImpl {
       numberOwner: this.genderNumberManager.getRefNumber(owner, params),
       genderOwned: this.genderNumberManager.getRefGender(owned, params),
       numberOwned: this.genderNumberManager.getRefNumber(owned, params),
+      personOwner: 3, //first and second person not implemented

```

Comment:
I think it is not useful:
- the parameter is optional
- `getDet` implementation in `LanguageGerman.ts` does not use it

Reply:
ok, I ake it optional in DetParams